content/applications/sales/sales/ebay_connector/manage.rst

```diff
@@ -2,49 +2,115 @@
 
-To use pictures in your listing, you need to add them as **Attachments** on the product template.
+To use images in the listing, another option is to add them as **Attachments** on the product
+template.
 
 Listing with variations
 =======================
 
-When the **use eBay** on a product with variations is checked and with **Fixed
-Price** as **Listing Type**, the eBay form is slightly different. In the
-variants array, you can choose which variant will be listed on eBay as well as
-set the price and the quantity for each variant.
+When the :guilabel:`Sell on eBay` is checked on a product containing variations with
+:guilabel:`Fixed Price` as :guilabel:`Listing Type`, the eBay form is slightly different. Go to the
+:guilabel:`Variants` tab to or click :guilabel:`Configure Variants` in the top menu to configure the
+variant settings. Pricing can be configured for each variation.
 
-.. image:: manage/manage02.png
-  :align: center
+Product Identifiers
+===================
+
+Products identifiers such as EAN, UPC, Brand or MPN are required in most of the eBay categories.
+
+EAN and UPC identifiers
+-----------------------
+
+The module manages the EAN and UPC identifiers with the :guilabel:`Barcode` field of the product
+variant. If the :guilabel:`Barcode` field is empty or is value is not valid, the EAN and UPC values
+will be set as 'Does not apply' as recommended by eBay.
 
 Listing with item specifics
-===========================
+---------------------------
 
-In order to add item specifics, you should create a product attribute with one
-value in the **Variants** tab on the product form.
+In order to add item specifics, one should create a product attribute with a single value in the
+:guilabel:`Variants` tab on the product form. Examples of item specifics include: `MPN` or `Brand`.
+The Brand and MPN values are working as item specifics and should be defined in the
+:guilabel:`Variants` tab on the product form. If these values are not set, 'Does not apply' will be
+used for the eBay listing.
 
-.. image:: manage/manage03.png
-  :align: center
+Process Invoices and Payments
+=============================
 
-Product Identifiers
-===================
+Posting payment
+---------------
+
+When eBay orders are placed they are always paid for up front, via the eBay site. At no point will
+users pay for items on eBay through Odoo. Therefore, once orders are synced into Odoo from eBay they
+are already paid for, and Odoo's invoice / payment functionality will not need to be utilized.
+However, invoices need to be created and marked as Paid to “close” the sale order.
+
+Users can opt to mass create and post invoices in batches. To do so, navigate to Quotations in the
+list view by going to :menuselection:`Sales app --> Orders --> Quotations`. In the upper right
+corner select the list view icon. Then check the boxes that invoices should be made for and go to
+the :guilabel:`Action` menu. Click on :guilabel:`Create Invoices`.
+
```

Comment:
totally optional, but maybe we could add a bit more visual guidance here? Since "list view" icon might not make sense to a new user
![image](https://github.com/odoo/documentation/assets/23611711/307918f4-141d-4220-be67-cdee2bc47cc2)